examples/Order/ProcessOrder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Exemples\Order;
+
+// Step 1: Require the library from your Composer vendor folder
+require_once '../../vendor/autoload.php';
+
+use MercadoPago\Client\Common\RequestOptions;
+use MercadoPago\Client\Order\OrderClient;
+use MercadoPago\Exceptions\MPApiException;
+use MercadoPago\MercadoPagoConfig;
+
+// Step 2: Set production or sandbox access token
+MercadoPagoConfig::setAccessToken("<ACCESS_TOKEN>");
+// Step 2.1 (optional - default is SERVER): Set your runtime enviroment from MercadoPagoConfig::RUNTIME_ENVIROMENTS
+// In case you want to test in your local machine first, set runtime enviroment to LOCAL
+MercadoPagoConfig::setRuntimeEnviroment(MercadoPagoConfig::LOCAL);
+
+// Step 3: Initialize the API client
+$client = new OrderClient();
+
+$orderClient = new OrderClient();
+$order_id = "<ORDER_ID>";
+try {
```

Comment:
client está sendo inicializado duas vezes
```suggestion
$orderClient = new OrderClient();
$order_id = "<ORDER_ID>";

try {
```